src/DetailsView/details-view-initializer.ts

```diff
@@ -194,7 +194,7 @@ if (isNaN(tabId) === false) {
 
             const fixInstructionProcessor = new FixInstructionProcessor();
 
-            const reportHtmlGeneratorV2 = new ReportHtmlGeneratorV2(
+            const reportHtmlGeneratorV2 = new ReportHtmlGeneratorImpl(
```

Comment:
Variable name update?

Reply:
Good catch. Though I got it before.
Updated.